test/Cpp/src/PhysicalConnection.lf

```diff
@@ -0,0 +1,31 @@
+/**
+*
+* @author Maiko Brants TU Dresden
+*
+* Test physical connections.
+*
+* Modeled after the C version of this test.
+**/
+target Cpp;
+reactor Source {
+    output out:int;
+    reaction(startup) -> out {=
+        out.set(42);
+    =}
+}
+reactor Destination {
+    input in:int;
+    reaction(in) {=
+        auto time{get_elapsed_physical_time()};
+        reactor::log::Info() << "Received " << *in.get() << " at logical time " << time.count() << ".";
+        if(time.count() <= 0LL) {
```

Comment:
It is better to compare with `reactor::TimePoint::zero()`

Reply:
I could not find zero() in TimePoint but reactor::Duration::zero() seems to do the job?

Reply:
Yep, right, that is what I meant ;)